src/applications/disability-benefits/all-claims/tests/submit-transformer.unit.spec.js

```diff
@@ -30,21 +25,36 @@ import {
 } from '../constants';
 
 describe('transform', () => {
-  it('should transform minimal data correctly', () => {
-    expect(JSON.parse(transform(formConfig, minimalData))).to.deep.equal(
-      transformedMinimalData,
-    );
-  });
-  it('should transform maximal data correctly', () => {
-    expect(JSON.parse(transform(formConfig, maximalData))).to.deep.equal(
-      transformedMaximalData,
-    );
-  });
-  it('should transform ptsd form upload data correctly', () => {
-    expect(
-      JSON.parse(transform(formConfig, minimalPtsdFormUploadData)),
-    ).to.deep.equal(transformedMinimalPtsdFormUploadData);
-  });
+  // Read all the data files
+  const dataDir = path.join(__dirname, './data/');
```

Comment:
Most of this PR is adding test data and moving around existing test data, but it's worth looking here.

This will automatically require us to create corresponding transformed data when we add new test data files.